controllers/vmi_controller.go

```diff
@@ -0,0 +1,212 @@
+package controllers
```

Comment:
I think need to give more meaningful name to this file, something like 'vmievacuation_controller.go